From 454465806fa663416533248efc499dc0a3b360f4 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 23 Sep 2016 11:53:59 -0700
Subject: [PATCH] IMGAPI-586 authkeys refactor for IMGAPI signature auth
 Reviewed by: Julien Gilli <julien.gilli@joyent.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 CHANGES.md    | 4 ++++
 lib/imgapi.js | 4 ++--
 package.json  | 2 +-
 3 files changed, 7 insertions(+), 3 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index 1cbae68..0e171bf 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,10 @@
 
 # sdc-clients Changelog
 
+## 9.5.1
+
+- IMGAPI-586: Change the endpoint URL for IMGAPI AdminReloadAuthKeys.
+
 ## 9.5.0
 
 - IMGAPI-579: client support for IMGAPI AdminReloadAuthKeys endpoint
diff --git a/lib/imgapi.js b/lib/imgapi.js
index 6d91122..e603170 100644
--- a/lib/imgapi.js
+++ b/lib/imgapi.js
@@ -480,7 +480,7 @@ IMGAPI.prototype.adminGetState = function adminGetState(callback) {
 
 
 /**
- * Tell IMGAPI to reload its auth keys (if relevant).
+ * Tell IMGAPI to reload its authkeys (if relevant).
  *
  * @param {Function} callback : `function (err, res)`
  */
@@ -488,7 +488,7 @@ IMGAPI.prototype.adminReloadAuthKeys = function adminReloadAuthKeys(callback) {
     var self = this;
     assert.func(callback, 'callback');
 
-    var path = self._path('/keys/reload');
+    var path = self._path('/authkeys/reload');
     self._getAuthHeaders(function (hErr, headers) {
         if (hErr) {
             callback(hErr);
diff --git a/package.json b/package.json
index a8f2422..ead590e 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdc-clients",
   "description": "Contains node.js client libraries for SDC REST APIs.",
-  "version": "9.5.0",
+  "version": "9.5.1",
   "homepage": "http://www.joyent.com",
   "repository": {
     "type": "git",
-- 
2.21.0

